concepts/05 UI Components/Gantt/05 Overview/10 Key Features.md

```diff
@@ -0,0 +1,41 @@
+- **Task Creation**    
+You can specify tasks in the Gantt [data source](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/tasks/#dataSource) or enable the component's [editing](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/editing/) tools to create tasks.
+
+- **Task Types**    
+Gantt supports tasks of multiple types:
```

Comment:
we can add a link to the task section here as well [tasks](Documentation/Guide/UI_Components/Gantt/Gantt_Elements/#Task)